src/Compilers/VisualBasic/Portable/Binding/Binder_ObjectInitializer.vb

```diff
@@ -905,6 +905,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                                           callerInfoOpt:=topLevelInitializer)
                 invocation.SetWasCompilerGenerated()
 
+                If (invocation.Kind = BoundKind.LateInvocation) Then
```

Comment:
>( [](start = 19, length = 1)

Parens are not neede in VB. Consider removing, given that the condition is very simple